src/app/pages/checkout-address/checkout-address-anonymous/checkout-address-anonymous.component.html

```diff
@@ -4,106 +4,133 @@
     <ish-error-message [error]="error" [toast]="false"></ish-error-message>
   </div>
 
-  <!--- login --->
+  <!-- login -->
   <div class="col-lg-6">
     <h2>{{ 'checkout.account.login.returning_customer.heading' | translate }}</h2>
     <ish-identity-provider-login></ish-identity-provider-login>
   </div>
-  <!-- checkout as guest -->
+
   <div class="col-lg-6">
-    <h2>{{ 'checkout.addresses.checkout_as_guest.heading' | translate }}</h2>
-    <p>{{ 'checkout.address.anonymous.guestcheckout.text' | translate }}</p>
-    <button
-      *ngIf="isAddressFormCollapsed"
-      class="btn btn-secondary"
-      type="button"
-      (click)="showAddressForm()"
-      [attr.aria-expanded]="!isAddressFormCollapsed"
-      aria-controls="collapseBasic"
-      data-testing-id="guest-checkout-button"
-    >
-      {{ 'checkout.addresses.checkout_as_guest.heading' | translate }}
-    </button>
+    <!-- registration -->
+    <div class="section">
+      <h2>{{ 'account.new_user.heading' | translate }}</h2>
+      <p>{{ 'account.create.benefit.text' | translate }}</p>
+      <ul>
+        <li>{{ 'account.create.benefit1.text' | translate }}</li>
+        <li>{{ 'account.create.benefit2.text' | translate }}</li>
+        <li>{{ 'account.create.benefit3.text' | translate }}</li>
+        <li>{{ 'account.create.benefit4.text' | translate }}</li>
+        <li>{{ 'account.create.benefit5.text' | translate }}</li>
+      </ul>
+      <a
+        class="btn btn-secondary"
+        routerLink="/register"
+        [queryParams]="{ returnUrl: '/checkout', cancelUrl: '/checkout' }"
```

Comment:
Why are you using both query params now?

Reply:
2 params are needed, because the return url and the cancel url might be different. In many cases the user has to be logged in to get navigated to the return url (e.g. the myAccount page, the wishlist page etc.) whereas the cancel url can be reached by an anonymous user (default = home).

Reply:
OK. Understood